src/css/spectrum/spectrum.css

```diff
@@ -13,7 +13,7 @@ License: MIT
     *display: inline;
     *zoom: 1;
     /* https://github.com/bgrins/spectrum/issues/40 */
-    z-index: 9999994;
+    z-index: 20000;
```

Comment:
This value was way to high for the Bootstrap tooltip (zindex: 30000).
Which one should be changed ?